packages/docs/docs/components/code-block.mdx

```diff
@@ -165,7 +165,8 @@ for future edits. Then call `edit` with an embedded `insert` call to add the new
 code.
 
 ```tsx
-import {CodeBlock, insert} from '@motion-canvas/2d/lib/components';
+import {insert} from '@motion-canvas/2d/lib/components';
```

Comment:
`insert` is also currently not exported from `'@motion-canvas/2d/lib/components`